docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -272,41 +275,113 @@ Where,
 
 We can refer to the outcome rate multiplied by (1 - PAF) as the "risk-deleted outcome rate."
 
-.. todo::
+Using RRs and PAFs from GBD
+---------------------------
 
-    Add a note about bias this introduces...
+While GBD reports RRs and PAFs, they are often not suitable for use in Vivarium models.
+This is because we typically model GBD causes as dynamic state-transition models,
+and are interested in applying RRs and PAFs to *transitions* in the model.
 
-        PAF relies on exposure in the population, not the "at-risk" group for the outcome. This bias is larger when the at-risk population is small relative to the total population.
+GBD reports RRs and PAFs for years of life lost (YLLs) and years lived with disability (YLDs).
+YLLs are the number of deaths (due to a cause) multiplied by the average remaining life expectancy
+(which is a constant for a given age and sex group).
+YLDs are the number of prevalent cases of a cause multiplied by the disability weight
+of that cause, which is a constant for a given cause.
+Therefore
```

Comment:
```suggestion
Therefore, since relative risk is computed as a ratio (and the PAF is computed from the RRs), the constant factors cancel out, and we get
```
Like you said in doc session, maybe add a bit more explanation?